qiskit/primitives/utils.py

```diff
@@ -25,8 +25,10 @@
 from qiskit.quantum_info import PauliList, SparsePauliOp, Statevector
 from qiskit.quantum_info.operators.base_operator import BaseOperator
 from qiskit.quantum_info.operators.symplectic.base_pauli import BasePauli
+from qiskit.utils.deprecation import deprecate_func
 
 
+@deprecate_func(since="1.2", additional_msg="There is no replacement for this.")
```

Comment:
@ElePT I'm wondering what is a good message if there is no replacement. Do you have any idea?
Three functions in this file don't have any replacement.
https://github.com/Qiskit/qiskit/pull/11490/files#r1445895211

Reply:
@t-imamichi Would [this](https://github.com/Qiskit/qiskit/pull/12575/files#diff-c789a3af03a0881f870c90d3dcdbd86c3a5a9d293deadbccbaa4c4437953d696R33) variant would be more helpful to the user?

Reply:
I wrote more details of replacements.